lib/Scheduler/enumerator.cpp

```diff
@@ -993,6 +993,10 @@ bool Enumerator::FindNxtFsblBrnch_(EnumTreeNode *&newNode) {
 #endif
 
     if (i == brnchCnt - 1) {
+      if (!(!getIsTwoPass() || getIsSecondPass())) {
+        return false;
+      }
```

Comment:
Recommend De Morgan's-izing this: `if (getIsTwoPass() && !getIsSecondPass()) {`. I think that's a little easier to understand, "If we are in two-pass mode but not in the second pass".

Reply:
Alternatively, factor out the `(!getIsTwoPass() || getIsSecondPass())` with the other `if` further down, so that this is `if (!Condition)` and the other is `if (Condition)`